src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/FalconElbow/FalconElbowConstants.java

```diff
@@ -0,0 +1,65 @@
+package edu.greenblitz.robotName.subsystems.arm.ElbowUtils.FalconElbow;
+
+import com.ctre.phoenix6.configs.*;
+import com.ctre.phoenix6.controls.MotionMagicDutyCycle;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.greenblitz.robotName.subsystems.arm.ElbowUtils.ElbowConstants;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+
+import static edu.greenblitz.robotName.subsystems.arm.ElbowUtils.ElbowConstants.*;
+
+public class FalconElbowConstants {
+
+    public static final int MOTOR_ID = 1;
+
+    public static final SimpleMotorFeedforward SIMPLE_MOTOR_FEED_FORWARD = new SimpleMotorFeedforward(kS, kV, kA);
+
+    public static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+
+    public static final MotionMagicConfigs MOTION_MAGIC_CONFIGS = new MotionMagicConfigs();
+
+    static {
+        MOTION_MAGIC_CONFIGS.MotionMagicAcceleration = 1;
+        MOTION_MAGIC_CONFIGS.MotionMagicCruiseVelocity = 2;
+        MOTION_MAGIC_CONFIGS.MotionMagicJerk = 4;
+    }
+
+    public static final ClosedLoopRampsConfigs CLOSED_LOOP_RAMPS_CONFIGS = new ClosedLoopRampsConfigs();
+
+    static {
+        CLOSED_LOOP_RAMPS_CONFIGS.DutyCycleClosedLoopRampPeriod = 1;
+        CLOSED_LOOP_RAMPS_CONFIGS.TorqueClosedLoopRampPeriod = 2;
+        CLOSED_LOOP_RAMPS_CONFIGS.VoltageClosedLoopRampPeriod = 3;
+    }
+
+    public static final CurrentLimitsConfigs CURRENT_LIMITS_CONFIGS = new CurrentLimitsConfigs();
+
+    static {
+        CURRENT_LIMITS_CONFIGS.StatorCurrentLimit = 1;
+        CURRENT_LIMITS_CONFIGS.StatorCurrentLimitEnable = true;
+        CURRENT_LIMITS_CONFIGS.SupplyCurrentLimit = 2;
+        CURRENT_LIMITS_CONFIGS.SupplyCurrentLimitEnable = true;
+        CURRENT_LIMITS_CONFIGS.SupplyCurrentThreshold = 3;
+        CURRENT_LIMITS_CONFIGS.SupplyTimeThreshold = 4;
+    }
+
+    public static final SoftwareLimitSwitchConfigs SOFTWARE_LIMIT_SWITCH_CONFIGS = new SoftwareLimitSwitchConfigs();
+
+    static {
+        SOFTWARE_LIMIT_SWITCH_CONFIGS.ForwardSoftLimitEnable = true;
+        SOFTWARE_LIMIT_SWITCH_CONFIGS.ForwardSoftLimitThreshold = FORWARD_ANGLE_LIMIT;
+        SOFTWARE_LIMIT_SWITCH_CONFIGS.ReverseSoftLimitEnable = true;
+        SOFTWARE_LIMIT_SWITCH_CONFIGS.ReverseSoftLimitThreshold = BACKWARD_ANGLE_LIMIT;
+    }
+
+    public static final TalonFXConfiguration TALON_FX_CONFIGURATION = new TalonFXConfiguration();
+
+    static{
+        TALON_FX_CONFIGURATION.MotionMagic = MOTION_MAGIC_CONFIGS;
+        TALON_FX_CONFIGURATION.ClosedLoopRamps = CLOSED_LOOP_RAMPS_CONFIGS;
+        TALON_FX_CONFIGURATION.CurrentLimits = CURRENT_LIMITS_CONFIGS;
+        TALON_FX_CONFIGURATION.SoftwareLimitSwitch = SOFTWARE_LIMIT_SWITCH_CONFIGS;
+    }
+
+    public static final MotionMagicDutyCycle MOTION_MAGIC_DUTY_CYCLE = new MotionMagicDutyCycle(0, true,FalconElbowConstants.SIMPLE_MOTOR_FEED_FORWARD.calculate(0), 0, true, true, true);
```

Comment:
move to the subsystem. this is not a constant, the FF should be added every time you set the voltage or else we will laways get a zero